docs/content/contribute/styleguide.md

```diff
@@ -0,0 +1,324 @@
+
+### Diagrams
+
+For diagrams (not bitmap images such as photos or scans) we much prefer SVG, as clean as you can make it, due to its scalability, adaptability and versatility.
+
+Like any web site, Hugo will also serve JPEG and PNG formats to browsers, but it will not be as easy to align colors and fonts. `alt` tagging is always wanted on any images called in as files, as described under [Images](#images).
+
+#### Mermaid diagrams
+
+For certain kinds of diagrams, our Hugo theme also supports inline [Mermaid notation](https://mermaid-js.github.io/mermaid/#/). A Hugo shortcode is used to demarcate input for Mermaid to interpret and plot, so for
+
+```
+{{</* mermaid */>}}
+pie
+    title Ice Cream Preferences
+    "Vanilla" : 50
+    "Chocolate Fudge" : 35
+    "Pistachio" : 10
+    "Peppermint" : 5
+{{</* /mermaid */>}}    
+```
+
+Hugo (with help from MermaidJS) produces
+
+{{< mermaid >}}
+pie
+    title Ice Cream Preferences
+    "Vanilla" : 50
+    "Chocolate Fudge" : 35
+    "Pistachio" : 10
+    "Peppermint" : 5
+{{< /mermaid >}}
+
+### Hugo and Markdown
+
+* [Hugo](https://gohugo.io/) - static site generator
+* [US Web Design System](https://designsystem.digital.gov/)
+* Our [implementation](https://pages.nist.gov/hugo-uswds-docs/) of USWDS in Hugo
+
+Keep in mind when editing Markdown files that Markdown, lacking a single external specification, varies from one implementation to another. Creating a file in Github using Github Markdown, then publishing it through the site, will *mostly* work. To see it as readers will see it, especially if using features of USWDS, you can preview your work as it is built and served by running a local copy of the site.
+
+#### Running the OSCAL site locally
+
+Running Hugo with its own server locally is a convenient way of previewing site contributions as they will actually appear. Not all content (especially if unproblematic Markdown) needs to be previewed, but it is a good way to intercept display problems early.
+
+This is documented in the repository [docs directory readme](https://github.com/usnistgov/OSCAL/tree/main/docs).
+
+From a Linux shell, use command
+
+```sh
+$ hugo server -v --debug --minify
+```
+
+to start the server on port 1313, viewable at `http://localhost:1313/OSCAL/`.
+
+Note that (as detailed in the readme) to browse the entire site with all contents, including generated contents, additional scripting may need to be run prior to starting the server.
+
+## Style
+
+### Accessibility
+
+Since materials for the web site are authored using generic encoding (Markdown) and produced with an automated production pipeline, we are able to encapsulate and manage page design within our build. This takes accessibility and accessibility compliance (such as ADA) out of the hands of authors, for the most part.
```

Comment:
See above. This could arguably at some point be separated out; for now we need a stake in the ground. (Metaphor without scare quotes.) Don't think a Style Guide today should be without some level of treatment.